collect_stata/write_json.py

```diff
@@ -149,105 +166,107 @@ def stats_number(elem, file_csv):
     }
 
 
-def uni_statistics(elem, file_csv):
-    """Call function to generate statistics depending on the variable type
+def uni_statistics(elem, data):
+    """Call function to generate statistics depending on the variable type.
 
-    Input:
-    elem: dict
-    file_csv: pandas DataFrame
+    Args:
+        elem (dict): Contains information of one variable.
+        data (pandas.DataFrame): Datatable of imported data.
 
-    Output:
-    statistics: OrderedDict
+    Returns:
+        statistics (OrderedDict):
+        Statistics for either categorical, nominal or numerical variables.
     """
 
     if elem["type"] == "cat":
 
-        statistics = stats_cat(elem, file_csv)
+        statistics = stats_cat(elem, data)
 
     elif elem["type"] == "string":
 
-        statistics = stats_string(elem, file_csv)
+        statistics = stats_string(elem, data)
 
     elif elem["type"] == "number":
 
-        statistics = stats_number(elem, file_csv)
+        statistics = stats_number(elem, data)
 
     else:
         statistics = dict()
```

Comment:
Your return type in the docstring does not match here.

Reply:
What do you mean? You could use mypy.

The docstring states that you return an "ordereddict", but in the else clause you create a dict.
The behave similarly. I guess you could say an "ordereddict is a dict". But not the other way around.